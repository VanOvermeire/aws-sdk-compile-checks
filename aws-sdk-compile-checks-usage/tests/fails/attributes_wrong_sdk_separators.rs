use aws_sdk_compile_checks_macro::required_props;

#[required_props(sdk = sqs/s3)]
async fn do_call() {
    // irrelevant
}

fn main() {}
