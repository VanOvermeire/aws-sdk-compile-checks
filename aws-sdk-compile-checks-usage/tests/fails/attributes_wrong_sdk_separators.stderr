error: expected one or more SDKs, separated by `,` after keyword `sdk`, e.g. `sdk = sqs,s3`
 --> tests/fails/attributes_wrong_sdk_separators.rs:3:27
  |
3 | #[required_props(sdk = sqs/s3)]
  |                           ^
